content/hi/edge-computing.md

```diff
@@ -0,0 +1,27 @@
+---
+title: एज-कंप्यूटिंग (Edge Computing)
+status: Completed
+category: प्रौद्योगिकी
+---
+
+एज-कंप्यूटिंग एक [डिस्ट्रिब्यूटेड सिस्टम](/distributed-systems/) का दृष्टिकोण है, जो कुछ स्टोरेज और कंप्यूटिंग क्षमता को प्राथमिक डेटा केंद्र से डेटा स्रोत में स्थानांतरित करता है।
+एकत्रित डेटा की गणना प्रसंस्करण और विश्लेषण के लिए केंद्रीकृत डेटा सेंटर में भेजने के बजाय स्थानीय स्तर पर की जाती है (उदाहरण के लिए, किसी कारखाने के फर्श पर, किसी स्टोर में, या पूरे शहर में)।
+ये स्थानीय प्रसंस्करण डिवाइस सिस्टम के एज का प्रतिनिधित्व करते हैं, जबकि डेटा सेंटर इसका केंद्र है।
+एज पर गणना किए गए आउटपुट को आगे की प्रक्रिया के लिए प्राथमिक डेटा सेंटर में वापस भेजा जाता है।
+एज-कंप्यूटिंग के उदाहरणों में कलाई गैजेट या कंप्यूटर शामिल हैं जो ट्रैफ़िक प्रवाह का विश्लेषण करते हैं।
+
+## समस्या
+
+पिछले दशक में, हमने अत्याधुनिक एज-डिवाइस (जैसे, मोबाइल फोन, स्मार्ट घड़ियाँ, या सेंसर) की बढ़ती संख्या देखी है।
+कुछ मामलों में, वास्तविक डेटा प्रोसेसिंग न केवल अच्छा है बल्कि महत्वपूर्ण भी है।
+आप उदाहरण के तौर पर सेल्फ-ड्राइविंग कारों के बारे में सोच सकते हैं।
+अब कल्पना करें कि कार के सेंसर से डेटा को वाहन में वापस भेजने से पहले प्रसंस्करण के लिए डेटा सेंटर में स्थानांतरित करना होगा ताकि यह उचित रूप से प्रतिक्रिया कर सके।
```

Comment:
`कार` would be better with `गाड़ी`

Reply:
Yes it is much better than गाड़ी

Reply:
Shouldn't we follow `गाड़ी` the same thing everywhere as changing `स्वचलित गाड़ियों`?